test/low_level/test_frame_info.py

```diff
@@ -59,17 +60,18 @@ def test_frame_info_hide_false():
     assert stat_profile_c.get_frame_info(frame) == stat_profile_python.get_frame_info(frame)
 
 
-def test_frame_info_with_classes():
-    instance = AClass()
+@pytest.mark.parametrize('test_function',
+                         ['get_frame_info_for_a_method',
+                          'get_frame_info_for_a_class_method',
+                          'get_frame_info_with_cell_variable'])
+def test_frame_info_with_classes(test_function):
+    if 'class' in test_function:
+        test_function = getattr(AClass, test_function)
+    else:
+        instance = AClass()
+        test_function = getattr(instance, test_function)
```

Comment:
The getattr stuff muddles me a bit, does this work? 

```suggestion
instance = AClass()

@pytest.mark.parametrize('test_function',
                         [instance.get_frame_info_for_a_method,
                          AClass.get_frame_info_for_a_class_method,
                          instance.get_frame_info_with_cell_variable])
def test_frame_info_with_classes(test_function):
```